security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -486,11 +486,16 @@ func (sc *SecretCache) generateSecret(ctx context.Context, token, resourceName s
 	}
 
 	startTime := time.Now()
-	retry := 0
-	backOffInMilliSec := initialBackOffIntervalInMilliSec
+	backoffBase := 1
 	var certChainPEM []string
 	for true {
-		certChainPEM, err = sc.fetcher.CaClient.CSRSign(ctx, csrPEM, exchangedToken, int64(sc.configOptions.SecretTTL.Seconds()))
+		// Add some jitter to avoid the thundering herd problem to CA.
+		backOffInMilliSec := rand.Intn(backoffBase * sc.configOptions.InitialBackoff)
+		log.Warnf("Will send CSR in %d millisec", backOffInMilliSec)
```

Comment:
Why is this Warnf in stead of Infof/Debugf?

Reply:
I followed the initial format. And I feel it actually makes sense to use Warnf since it is actually a warning :), i believe when you turn on Debug mode, this warning message will show up

Reply:
Yeah, that's not a big deal :)